xonsh/prompt/cwd.py

```diff
@@ -89,3 +89,12 @@ def _dynamically_collapsed_pwd():
                 0 : int(target_width) - len(truncature_char)
             ] + truncature_char
     return full
+
+
+def _add_dir_slash(cwd_dir):
```

Comment:
I think this is usually implemented using `os.path.join(cwd_dir, "")`. No extra function needed.

Reply:
Thanks for the suggestion. That makes things simpler.